config/locales/en.yml

```diff
@@ -494,3 +506,5 @@ en:
         trn: "Teacher reference number (TRN)"
         full_name: "Full name"
         national_insurance_number: "National Insurance number (optional)"
+
+        date_of_birth: What is your date of birth?
```

Comment:
@mattonfoot I _think_ this is the error message that has its translation missing error read out in the clip you sent me.